files/system/river/etc/river/init

```diff
@@ -177,6 +177,9 @@ riverctl spawn "/usr/libexec/xfce-polkit"
 riverctl spawn waybar
 riverctl spawn kanshi
 riverctl spawn nm-applet
+riverctl spawn "swaybg --image /usr/share/backgrounds/f41/default/f41-01-night.png"
+riverctl spawn "/usr/libexec/xdg-desktop-portal-wlr"
+riverctl spawn "/usr/libexec/xdg-desktop-portal"
```

Comment:
Is this required? `xdg-desktop-portal.service` is a user service that should take care of this.

On hyprland I also have `xdg-desktop-portal-hyprland.service`. Is there another for `-wlr`? Are they not being launched when you log in?

Reply:
These units are required by `graphical-session.target`, so if you're logging into a tty and then spawning river manually, they may not be launched.

Reply:
commits have been updated, time for a test